src/DataTool.php

```diff
@@ -200,7 +205,7 @@ public function generateData()
      *
      * @return string
      */
-    public function generateId()
+    public function generateId($include_cstm = false)
```

Comment:
Let's use cameCaps syntax for variables, it's allowed right now for PSR2 code standard, but restricted in others.